ui/packages/app/src/components/Dataset/Dataset.svelte

```diff
@@ -61,36 +71,30 @@
 	);
 </script>
 
-<div
-	id={elem_id}
-	class="mt-4 inline-block max-w-full text-gray-700 w-full"
-	class:!hidden={!visible}
->
-	<div class="text-xs mb-2 flex items-center text-gray-500">
+<div id={elem_id} class="wrap" class:hide={!visible}>
+	<div class="label">
 		<svg
 			xmlns="http://www.w3.org/2000/svg"
 			xmlns:xlink="http://www.w3.org/1999/xlink"
 			aria-hidden="true"
 			role="img"
-			class="mr-1"
 			width="1em"
 			height="1em"
 			preserveAspectRatio="xMidYMid meet"
 			viewBox="0 0 32 32"
-			><path
+		>
+			<path
 				fill="currentColor"
 				d="M10 6h18v2H10zm0 18h18v2H10zm0-9h18v2H10zm-6 0h2v2H4zm0-9h2v2H4zm0 18h2v2H4z"
-			/></svg
-		>
+			/>
+		</svg>
 		{label}
 	</div>
 	{#if gallery}
-		<div class="gr-samples-gallery">
+		<div class="gallery">
 			{#each selected_samples as sample_row, i}
-				<!-- {@const x = component_map[]} -->
-
 				<button
-					class="group rounded-lg"
+					class="button"
```

Comment:
I think this removes the border around the example when there are only examples for one component?

Is that intentional?

### This PR
![image](https://user-images.githubusercontent.com/41651716/210423094-ac89bf59-ea0b-42c0-827a-85d2f2828d0b.png)

### Main
![image](https://user-images.githubusercontent.com/41651716/210423729-a89a734a-771d-480f-913e-18671eecaf29.png)

Reply:
Ah, no this is a bug. I did see this and forgot to fix. Good catch.